app/views/components/_active_topic.html.erb

```diff
@@ -5,8 +5,16 @@
       <%= @active_current_topic.body.html_safe %>
     </div>
     <div>
-      <a href="#" data-key="<%= @active_current_topic.key %>">Skryť správu</a>
+      <a href="#" class="js__active-topic-close">Skryť správu</a>
     </div>
   </div>
   </div>
 </div>
+<script src="https://cdn.jsdelivr.net/npm/js-cookie@rc/dist/js.cookie.min.js"></script>
```

Comment:
Toto asi hodme do webpacker alebo niekam. Nie som uplny fanda nacitavania z random CDN bez integrity hash.

Reply:
oki hodim to localne niekam

Reply:
hotovo :)